chef_master/source/_templates/nav-docs.html

```diff
@@ -63,8 +63,19 @@
           },
           {
             "title": "Licensing",
-            "hasSubItems": false,
-            "url": "/chef_license.html"
+            "hasSubItems": true,
+            "subItems": [
+              {
+                "title": "About Licensing",
+                "hasSubItems": false,
+                "url": "/chef_license.html"
+              },
+              {
+                "title": "Accepting the Chef License",
```

Comment:
This title is being cut off at the end on the website. Would "Accepting License" be shorter yet accurate title?